docs/_static/css/custom.css

```diff
@@ -0,0 +1,300 @@
+    format("opentype");
+  font-weight: 300;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c898dc6c0c107af7c2ad_Graphik-Regular.otf")
+    format("opentype");
+  font-weight: 400;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c8986a906d68f93257ca_Graphik-BoldItalic.otf")
+    format("opentype");
+  font-weight: 700;
+  font-style: italic;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c898429c1a00eef87060_Graphik-Medium.otf")
+    format("opentype");
+  font-weight: 500;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c8986a906d7ed13257cd_Graphik-MediumItalic.otf")
+    format("opentype");
+  font-weight: 500;
+  font-style: italic;
+}
+@font-face {
+  font-family: "Graphik super";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c89878cf530ecd82d273_Graphik-Super.otf")
+    format("opentype");
+  font-weight: 400;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c898dc6c0c0f07f7c2ae_Graphik-SemiboldItalic.otf")
+    format("opentype");
+  font-weight: 600;
+  font-style: italic;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c898429c1aa41cf87061_Graphik-Semibold.otf")
+    format("opentype");
+  font-weight: 600;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c89878cf536aa782d274_Graphik-Thin.otf")
+    format("opentype");
+  font-weight: 100;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c898429c1aed84f87063_Graphik-RegularItalic.otf")
+      format("opentype"),
+    url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c898429c1a44ccf87062_Graphik-SuperItalic.otf")
+      format("opentype");
+  font-weight: 400;
+  font-style: italic;
+}
+@font-face {
+  font-family: "Graphik";
+  src: url("https://assets.website-files.com/5dc3b47ddc6c0c2a1af74ad0/5dc3c8986a906d79de3257d2_Graphik-ThinItalic.otf")
+    format("opentype");
+  font-weight: 100;
+  font-style: italic;
+}
+
+:root {
+  --main-bg-color: #ffffff;
+
+  --note-color: #e4f5ff; /* pale blue */
+  --note-title: #1c83e1; /* vivid blue */
+
+  --sidefont-color: #727377; /*darkest gray */
+
+  --tip-color: #dffde9; /* pale green */
+  --tip-title: #21c354; /* vivid green */
+
+  --heading-color: #ff4b4b; /* salmon */
+
+  --st-black: #262730;
```

Comment:
0.9% of our visits to the docs in the past 3 months were from any version of Internet Explorer. I think option 1 is preferable to 2, because adding steps to the Makefile seems excessive for values that shouldn't change frequently (i.e. I put the variables there for ease of development, but when development is over, they don't add much value)

Reply:
Heh I just did option 1, but via a find-and-replace script.

It does adds steps to the makefile, but (1) it makes life easier while developing, and (2) the script doesn't have any dependencies so we don't have to install anything.

Reply:
Another option would be to run the script _and commit the resulting CSS file_.

Reply:
I'd rather commit the CSS file, as I don't really use the make file as part of my development